esi_leap/db/sqlalchemy/api.py

```diff
@@ -344,10 +351,13 @@ def contract_update(contract_uuid, values):
     return contract_ref
 
 
-def contract_destroy(contract_uuid):
```

Comment:
I don't think we should remove these functions entirely; I'd leave these in the object and db layers.

Reply:
So we will have both the cancel and destroy functions? And just let DELETE call only the cancel?

Reply:
Yep, exactly.

Reply:
I have re-added the destroy functions.